Strongly recommend competitors to turn off notifications and calls (#1024)
thewca/wca-regulations: wca-regulations.md

        - 2i2a) Each camera monitor must be blank or out of sight of the competitor (see [Regulation A5b](regulations:regulation:A5b)).
        - 2i2b) The competitor must not interact with (e.g. operate, hold, wear) any active camera. Exception: the competitor may wear a camera mounted on their head, as long as it is out of their sight and it is clear that they are not interacting with it (apart from wearing it).
    - 2i3) The competitor may have a cell phone in their pocket, as long as it is clear that they are not otherwise interacting with it.

>     - 2i4) Competitors should turn off all cell phone notifications while competing to avoid disturbing themselves or others.

Reviewer: ```suggestion
    - 2i4) Competitors should turn off all cell phone notifications while competing to avoid disturbing the competition.
```
